lib/msf/core/exploit/format/webarchive.rb

```diff
@@ -4,6 +4,7 @@
 # installing extensions from extensions.apple.com.
 #
 module Msf
```

Comment:
FWIW, I can see a pro to using the nested syntax: you know exactly what namespaces you're dealing with, and if you use ```module``` where you want ```class``` (like in the line below), your code will break, and you'll know it.